Sources/StreamChatUI/ChatChannelList/Tests/ChatChannelListVC_SnapshotTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelListVC_SnapshotTests: XCTestCase {
+    var view: ChatChannelListItemView!
+    var vc: ChatChannelListVC!
+    var mockedChannelListController: ChatChannelListController_Mock<NoExtraData>!
+    
+    override func setUp() {
+        super.setUp()
+        UIConfig.default.channelList.channelListItemSubviews.avatarView = MockAvatarView.self
+        UIConfig.default.currentUser.currentUserViewAvatarView = MockCurrentChatUserAvatarView.self
+        vc = ChatChannelListVC()
+        mockedChannelListController = ChatChannelListController_Mock.mock()
+        vc.controller = mockedChannelListController
+    }
+    
+    func test_populated() {
```

Comment:
This just a nit since this is WIP, just not to forget...Probably it would be better to create some convention Or use something like `test_chatChannelList_isPopulated` as you can see with the rest of the tests... Or somehow start `test_snapshotTest_...`

Reply:
The snapshots are saved in folders with the name of the view controller already. From what I've seen, this is the pattern used in Snapshot tests, but I also prefer putting the name of what we are testing in the function name, even if in the folder/picture path it will be a bit repetitive, so I will use `test_chatChannelList_isPopulated`, thanks for the suggestion 👌